hikari/api/rest.py

```diff
@@ -177,9 +177,6 @@ async def fetch_channel(
         hikari.errors.RateLimitTooLongError
             Raised in the event that a rate limit occurs that is
             longer than `max_rate_limit` when making a request.
-        hikari.errors.RateLimitTooLongError
-            Raised in the event that a rate limit occurs that is
-            longer than `max_rate_limit` when making a request.
```

Comment:
This as probably been copy pasted to other places, did you do a quick search to check?

Reply:
I was not able to find any duplicates in hikari.api using this regex.
`hikari.*RateLimitToo.*\n.*\n.*\n\s+hikari.*RateLimitToo`.

Reply:
While my previous statement here may have been true, there were some instances of this in hikari.guilds. Resolved now and force pushed.